api/scholarqa/rag/retrieval.py

```diff
@@ -20,13 +20,16 @@ def rerank(self, query: str, retrieved_ctxs: List[Dict[str, Any]]) -> List[Dict[
 
 
 class PaperFinder(AbsPaperFinder):
+    snippet_srch_fields = ["text", "snippetKind", "snippetOffset", "section", "annotations.sentences", "annotations.refMentions",
+                           "annotations.sentences.start","annotations.sentences.end"]
```

Comment:
snippet search end point fields explicitly to be requested

Reply:
If we're including `"annotations.sentences.start","annotations.sentences.end"`, do we also need `"annotations.sentences"`?

Reply:
good catch! must have added .sentences while testing out the end point... I'll get rid of it